modules/gapi/samples/pipeline_modeling_tool/pipeline.hpp

```diff
@@ -40,35 +40,45 @@ std::string PerfReport::toStr(bool expand) const {
     return ss.str();
 }
 
+class StopCriteria {
```

Comment:
Probably over design. 
On the other hand what if another `stop criteria` will be needed (e.g N iters, but no more than Xms).

Reply:
I believe it is Criterion, not Criteria (plural form)

Reply:
Done